src/clarity-angular/data/datagrid/providers/hideable-column.service.ts

```diff
@@ -192,9 +192,9 @@ export class HideableColumnService {
      * @returns HideableColumn
      *
      */
-    public getColumnById( id: string ): DatagridHideableColumn {
+    public getColumnById( id: string ): any { // Can't be DatagridHideableColumn b/c we don't always return something.
```

Comment:
A wat? (j/k) I've seen them a couple of times but forgot all about them here. 
It absolutely makes sense. I'll push an update.